rsmas_logging.py

```diff
@@ -0,0 +1,71 @@
+import logging
```

Comment:
This abstraction currently isn't a subset of `logging` nor a superset of `logging`. Instead it depends on `logging` and users still need to import `logging` in order to deal with logging levels. This seems to be a messy state for this abstraction to be in.

What about adding logging levels to `rsmas_logging` (i.e. `DEBUG`, `INFO`, `WARNING`, etc)? `rsmas_logging` would then be a complete wrapper around `logging`.

Reply:
I agree. Good suggestion.